libsync/NodeTimeMaintenance.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+ * @CopyRight:
+ * FISCO-BCOS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FISCO-BCOS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FISCO-BCOS.  If not, see <http://www.gnu.org/licenses/>
+ * (c) 2016-2020 fisco-dev contributors.
+ */
+/**
+ * @brief : maintain the node time
+ * @file: NodeTimeMaintenance.cpp
+ * @author: yujiechen
+ * @date: 2020-06-12
+ */
+
+#include "NodeTimeMaintenance.h"
+#include <libsync/SyncMsgPacket.h>
+
+using namespace dev::sync;
+
+NodeTimeMaintenance::NodeTimeMaintenance()
+  : m_node2TimeOffset(std::make_shared<std::map<dev::h512, int64_t>>())
+{}
+
+void NodeTimeMaintenance::tryToUpdatePeerTimeInfo(SyncStatusPacket::Ptr _peerStatus)
+{
+    auto status = std::dynamic_pointer_cast<SyncStatusPacketWithAlignedTime>(_peerStatus);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'status' is assigned a value that is never used.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5697841)